Makefile

```diff
@@ -525,29 +525,34 @@ kvm_in_docker:
 	$(call DOCKER,$(KVM_BUILD_IMAGE),/usr/bin/make out/docker-machine-driver-kvm2 COMMIT=$(COMMIT))
 
 .PHONY: install-kvm-driver
-install-kvm-driver: out/docker-machine-driver-kvm2
+install-kvm-driver: out/docker-machine-driver-kvm2  ## Install KVM Driver
 	mkdir -p $(GOBIN)
 	cp out/docker-machine-driver-kvm2 $(GOBIN)/docker-machine-driver-kvm2
 
 .PHONY: release-kvm-driver
-release-kvm-driver: install-kvm-driver checksum
+release-kvm-driver: install-kvm-driver checksum  ## Release KVM Driver
 	gsutil cp $(GOBIN)/docker-machine-driver-kvm2 gs://minikube/drivers/kvm/$(VERSION)/
 	gsutil cp $(GOBIN)/docker-machine-driver-kvm2.sha256 gs://minikube/drivers/kvm/$(VERSION)/
 
 site/themes/docsy/assets/vendor/bootstrap/package.js:
 	git submodule update -f --init --recursive
 
-# hugo for generating site previews
 out/hugo/hugo:
 	mkdir -p out
 	test -d out/hugo || git clone https://github.com/gohugoio/hugo.git out/hugo
 	(cd out/hugo && go build --tags extended)
 
-# Serve the documentation site to localhost
 .PHONY: site
-site: site/themes/docsy/assets/vendor/bootstrap/package.js out/hugo/hugo
+site: site/themes/docsy/assets/vendor/bootstrap/package.js out/hugo/hugo ## Serve the documentation site to localhost
 	(cd site && ../out/hugo/hugo serve \
 	  --disableFastRender \
 	  --navigateToChanged \
 	  --ignoreCache \
 	  --buildFuture)
+
+.PHONY: help
+help:
+	@echo "\e[1mAvailable targets for minikube ${VERSION}\e[21m"
```

Comment:
These don't seem to be working here:

```console
$ make help
\e[1mAvailable targets for minikube v1.4.1\e[21m
\e[1m--------------------------------------\e[21m
...
```

The `printf` version works (listed in cyan), so maybe use that (for bold) instead ?

```sh
printf "\033[1mAvailable targets for minikube ${VERSION}\033[21m\n"
```